test/addons/openssl-client-cert-engine/testengine.cc

```diff
@@ -19,6 +19,12 @@
 #define AGENT_KEY           "test/fixtures/keys/agent1-key.pem"
 #define AGENT_CERT          "test/fixtures/keys/agent1-cert.pem"
 
+#ifdef _WIN32
```

Comment:
Assuming `NODE_EXTERN` does what you want, you could even
```C++
#include <node.h>  // for NODE_EXTERN
```
at the top, use it here, and not need to define another directive that serves a very similar purpose.